covidWatch/home.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title>COVIDWatch Homepage</title>
+</head>
+<body>
+	<a href="#"><img src="images/logo.png"></a>
+  		<a href="#">HOME</a>
```

Comment:
Let's align this. You can use something like https://www.freeformatter.com/html-formatter.html to get a good formatting. Ideally your IDE should also have a formatting option if you already use one.